docs/_data/table/en/common-configuration.yml

```diff
@@ -313,9 +321,11 @@ fs.cos.secret.key:
 fs.cos.socket.timeout:
   'The timeout of COS socket.'
 fs.gcs.accessKeyId:
-  'The access key of GCS bucket.'
+  'The access key of GCS bucket.This property keyis only valid when alluxio.underfs.gcs.version=1'
+fs.gcs.credential.path:
+  'The json file path of Google application credentials. This property key is only valid when alluxio.underfs.gcs.version=2'
 fs.gcs.secretAccessKey:
-  'The secret key of GCS bucket.'
+  'The secret key of GCS bucket.This property keyis only valid when alluxio.underfs.gcs.version=1'
```

Comment:
no space between the two sentences? mistake in the underlying javadoc?

Reply:
Oh dannn... it's built on my local maven build, never mind